components/collection/activity/Activity.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="is-flex">
+    <SidebarFilter />
+    <div ref="wrapper" class="w-full mt-5">
+      <div v-if="tablet">
+        <div class="columns">
+          <div class="column is-two-thirds">
+            <ActivityChart :events="events" />
+          </div>
+          <div class="column">
+            <OwnerInsights :owners="owners" :flippers="flippers" />
+          </div>
+        </div>
+        <BreadcrumbsFilter />
+      </div>
+      <div v-else>
+        <div class="is-flex is-flex-direction-column gap">
+          <OwnerInsights :owners="owners" :flippers="flippers" />
+          <div class="max-width">
+            <ActivityChart :events="events" />
+          </div>
+        </div>
+      </div>
+      <hr class="mb-40" :class="{ 'my-40': !isBreadCrumbsShowing }" />
+      <Events :events="sortedEventsWithOffersDesc" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import ActivityChart from './Chart.vue'
```

Comment:
Please use relative imports (starting with @)

Reply:
but Actvity.vue and Chart (renames to ActivityChart to be more clear) are in the same dir.
isn't this acceptable in that case?